### ISSUE TITLE
Naming a new world is broken

### ISSUE THREAD

**Issue by linusha** [LivelyKernel/lively.next#1108]
Creating a named world is broken (after entering a name and submitting it)

![image](https://github.com/LivelyKernel/lively.next/assets/14252419/6002c96c-45ea-44a0-82a7-2cd4e7eb9efa)

![image](https://github.com/LivelyKernel/lively.next/assets/14252419/1c98ae00-cf46-4320-9025-6dfb1b184254)

**Comment by rickmcgeer** [comment]
I am now seeing the following error when opening an _existing_ project:
```
I'm `fs` modules
bootstrap-9c3c6db6.js:634 Cannot read properties of undefined (reading 'REGISTRY')TypeError: 

    at Window.wrapModuleResolution (http://127.0.0.1:9011/lively.modules/src/system.js:40:25)
    at Window.prepareSystem (http://127.0.0.1:9011/lively.modules/src/system.js:258:21)
    at execute (http://127.0.0.1:9011/lively.modules/index.js:353:184)
    at ensureEvaluated (http://127.0.0.1:9011/lively.next-node_modules/systemjs/dist/system.src.js:3363:26)
    at Object.execute (http://127.0.0.1:9011/lively.next-node_modules/systemjs/dist/system.src.js:3513:13)
    at doDynamicExecute (http://127.0.0.1:9011/lively.next-node_modules/systemjs/dist/system.src.js:766:25)
    at link (http://127.0.0.1:9011/lively.next-node_modules/systemjs/dist/system.src.js:964:20)
    at doLink (http://127.0.0.1:9011/lively.next-node_modules/systemjs/dist/system.src.js:623:7)
    at updateLinkSetOnLoad (http://127.0.0.1:9011/lively.next-node_modules/systemjs/dist/system.src.js:669:18)
    at http://127.0.0.1:9011/lively.next-node_modules/systemjs/dist/system.src.js:485:11
system.js:40 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'REGISTRY')
    at Window.wrapModuleResolution (system.js:40:25)
    at Window.prepareSystem (system.js:258:21)
    at execute (index.js:353:184)
    at ensureEvaluated (system.src.js:3363:26)
    at Object.execute (system.src.js:3513:13)
    at doDynamicExecute (system.src.js:766:25)
    at link (system.src.js:964:20)
    at doLink (system.src.js:623:7)
    at updateLinkSetOnLoad (system.src.js:669:18)
    at system.src.js:485:11
```
This appears similar to the new project issue I had over the weekend.  Running `install.sh` doesn't help

**Comment by merryman** [comment]
@rickmcgeer No, that error looks quite different. It looks like this is very, very likely a caching issue inside your browser, since I see your browser is utilizing a `system.src.js` file which we do not use any more. This can only happen if you previously loaded a lively.next session based of a very outdated version of lively.next. Please try to clear your browser cache and local storage and load again. The easiest way is to just open lively inside an icognito window.

**Comment by rickmcgeer** [comment]
@merryman I reloaded in an incognito window and saw the same error.  I then did
```
$ git checkout origin main
$ git pull
$ ./install.sh
```
to ensure that I was up-to-date, and then loaded in an incognito window.  Same issue.

**Comment by linusha** [comment]
Closed by 5145e4ec90c971d0830dcdeb4fa1cdd668af71b0.
